sdcm/sct_events/base.py

```diff
@@ -59,6 +60,19 @@ def __set_name__(self, owner: Type[SctEvent], name: str) -> None:
         self[owner.__name__] = owner  # add owner class to the registry.
 
 
+@dataclass
+class SubContext:
+    event_id: str
+    base: str
+    node: str
+    timestamp: float
+    nemesis_name: str
```

Comment:
this is very specific to one type of Event, maybe there should be a dict of extra parameters ?

or add a `dict` function to the BaseEvent ? and remove the `SubContext` ?

Reply:
removed